docs/02-admin/01-installation/bare_metal.md

```diff
@@ -41,22 +41,16 @@ On **Debian 12** or later, you can install the latest PHP package repository (th
 sudo sh -c 'echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/php.list'
 ```
 
-You can choose between PHP 8.2 or 8.3, but it is recommended to use PHP 8.3.
-
-Install _PHP 8.2_ with some important PHP extensions:
-
-```bash
-sudo apt-get update
-sudo apt-get install php8.2 php8.2-common php8.2-fpm php8.2-cli php8.2-amqp php8.2-bcmath php8.2-pgsql php8.2-gd php8.2-curl php8.2-xml php8.2-redis php8.2-mbstring php8.2-zip php8.2-bz2 php8.2-intl php8.2-bcmath -y
-```
-
-Or install _PHP 8.3_ with PHP extensions:
+Install _PHP 8.3_ with PHP extensions:
 
 ```bash
 sudo apt-get update
 sudo apt-get install php8.3 php8.3-common php8.3-fpm php8.3-cli php8.3-amqp php8.3-bcmath php8.3-pgsql php8.3-gd php8.3-curl php8.3-xml php8.3-redis php8.3-mbstring php8.3-zip php8.3-bz2 php8.3-intl php8.3-bcmath -y
 ```
 
+> [!NOTE]
+> If you are upgrading to PHP 8.3 from an older version, please re-review the [PHP configuration]("#php") section of this guide as existing `ini` settings are NOT copied to new versions.
```

Comment:
```suggestion
> If you are upgrading to PHP 8.3 from an older version, please re-review the [PHP configuration]("#php") section of this guide as existing `ini` settings are NOT automatically copied to new versions. Additionally review which php-fpm version is configured in your nginx site
```
The "automatically" is important I think. The other new sentence might not be what we want, but I had `php-8.2-fpm` hard coded in there and I think most of the people coming from kbin will have that